### ISSUE TITLE
support flex layout like in web

### ISSUE THREAD

**Issue by sgf** [modern-forms/Modern.Forms#77]
support flex layout like in web

**Comment by neoxeo** [comment]
Hi, sgf 

You can have a look to this other project if you need more possibilities :
https://github.com/alternetsoft/AlternetUI

**Comment by jpobst** [comment]
I don't think this is something Modern.Forms will tackle. The core library will likely stick to the established Winforms layouts like anchor/fill, grid layout, and flow layout.

However someone could write a FlexLayoutPanel and provide it as a NuGet package to use.
